cpp/farm_ng/core/logging/logger.h

```diff
@@ -198,7 +199,7 @@ inline StreamLogger& defaultLogger() {
               rhs),                                                          \
           __FILE__,                                                          \
           __LINE__,                                                          \
-          __func__ FARM_MAYBE_VARGS(__VA_ARGS__)(__VA_ARGS__));              \
+          __func__);                                                         \
```

Comment:
This is removing context from all our logs and asserts in downstream dependencies if I'm. not mistaken.

We should either create a configurable log macro as we discussed or get it to work as is. (See passing CI in downstream project as documentation what build configuration is needed ...)

Reply:
Can you share the compile flags used for the python workflow when it is failing?

https://stackoverflow.com/questions/63740056/variadic-macros-with-zero-arguments-doesnt-compile-even-with-va-args

Reply:
compiling with ``--std=gnu++17`` worked rather than ``--std=c++17`` thank you @strasdat .  I've reverted these changes to logger.h.